src/bpftrace.cpp

```diff
@@ -45,6 +45,17 @@ constexpr char CHILD_EXIT_QUIETLY = '\0';
 constexpr char CHILD_GO = 'g';
 
 int format(char * s, size_t n, const char * fmt, std::vector<std::unique_ptr<IPrintable>> &args) {
+  // Args have been made safe for printing by now, so replace nonstandard format
+  // specifiers with %s
+  std::string str = std::string(fmt);
+  size_t start_pos = 0;
+  while ((start_pos = str.find("%r", start_pos)) != std::string::npos)
+  {
+    str.replace(start_pos, 2, "%s");
+    start_pos += 2;
+  }
+  fmt = strdup(str.c_str());
```

Comment:
Don't think the dup is necessary, right?
```suggestion
  fmt = str.c_str();
```